src/citrine/_rest/collection.py

```diff
@@ -13,6 +13,8 @@
 # have subclasses override the create method.
 CreationType = TypeVar('CreationType', bound='Resource')
 
+DEFAULT_PER_PAGE = 20
```

Comment:
Does anyone else think it makes sense to have a smaller default page size for citrine-python? 20ish makes sense for the UI, but I imagine for most citrine-python use cases, we will just end up iterating over the entire result set anyway, meaning we'll need more requests with a small page size.

In the cursor-paging function, I default to `100`, but that wasn't a collective decision, really. cc: @mcgalcode @djack

Reply:
I don't have a strong opinion on this.  I just selected this to be the same as the API.  I'm happy to use 100.